mirage/apt/apt_inputs.py

```diff
@@ -334,7 +334,7 @@ def get_pointing_info(self, file, propid):
                             observation_label.append(obslabel)
                             observation_number.append(obsnum)
                             visit_number.append(visitnum)
-                            vid = str(propid) + obsnum + visitnum
+                            vid = "{}{}{}".format(str(propid).zfill(5), obsnum, visitnum)
```

Comment:
A more "pythonic" way of doing this would be to specify the length of the prop_id in the string formatter. For example:

`'{0:05d}'.format(757)` returns `00757`

Reply:
I originally had it that way, but was getting errors that I didn't understand...maybe because after printing the updated value I was intentionally crashing the code, but was within a try statement. Let me try again...

Reply:
Slightly modified version. `propid` is already a string coming out of the xml. For some reason I couldn't combine the `0:05d` format code with the others, like this:
`vid = "{0:05d}{}{}".format(int(propid), obsnum, visitnum)`
I had to have it on its own separate line to avoid an error.